groundstation/static/js/components/AddUser.jsx

```diff
@@ -0,0 +1,210 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from "@material-ui/core/Checkbox";
+import Paper from '@material-ui/core/Paper';
+
+
+class AddUser extends Component {
+    constructor() {
+        super();
+        this.state = {
+            username: null,
+            password: null,
+            password2: null,
+            error_message : '',
+            newUserIsAdmin: false,
+            success_message: ''
+        }
+        this.handleUsernameChange = this.handleUsernameChange.bind(this);
+        this.handlePasswordChange = this.handlePasswordChange.bind(this);
+        this.handlePassword2Change = this.handlePassword2Change.bind(this);
+    }
+
+
+
+    handleAddUser() {
+        event.preventDefault();
+        if (this.state.password !== this.state.password2) {
+            this.setState({
+                error_message: "Passwords do not match.",
```

Comment:
```suggestion
                error_message: 'Passwords do not match.',
```
Single-quotes for javascript, use double-quotes for HTML attributes (for consistency).
This should be the only case of this though everything else is all good.